.github/workflows/amarna.yml

```diff
@@ -0,0 +1,26 @@
+name: Amarna Analysis
+on:
+  push:
+    branches:
+      - develop
+      - main
```

Comment:
Any reason this is only run on `develop` & `main`? we should use `on: [push, pull_request]`